README.md

```diff
@@ -195,3 +195,14 @@ const store = createStore([
   process.env.NODE_ENV === 'production' && require('storeon/logger')
 ])
 ```
+
+### Debug
+
+Storeon supports debugging with [Redux DevTools Extension](https://github.com/zalmoxisus/redux-devtools-extension)
+
+```js
+import createStore from 'storeon'
+import devtools from 'storeon/devtools'
```

Comment:
It will put `storeon/devtools` to production build too. We need to use a method from the logger section above.